### PR TITLE
Line through points in P^n

### PR DESCRIPTION
**Pull request by MatthiasHu**
Prove that there (merely) exists a line through any two distinct points in projective space.

### CODE REVIEW

**Review by felixwellen — CHANGES_REQUESTED** [review]
Hurray! This is really nice. I only have requests concerning the documentation.

File: SyntheticGeometry/ProjectiveSpace/LineThroughPoints.lagda.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+  line-hits-point-1 : line-through-points (p one) ≡ [ b , b≠0 ]
+  line-hits-point-1 = cong [_] (Σ≡Prop (λ _ → isProp→ isProp⊥) ((
+    ((0r ⋆ a) + (1r ⋆ b))  ≡⟨ cong₂ _+_ (⋆AnnihilL _) (⋆IdL _) ⟩
+    (0m + b)               ≡⟨ +IdL _ ⟩
+    b                      ∎)))
+```
+
+If we are given distinct points (but not representatives),
+we can still derive the (mere) existence of a line through these points.
+
```
[the remark this answers is not in the document]
Comment (MatthiasHu) [reply SyntheticGeometry/ProjectiveSpace/LineThroughPoints.lagda.md:232]: True... I will try to reformulate.

File: SyntheticGeometry/ProjectiveSpace/LineThroughPoints.lagda.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+  ([a]≠[b] : ¬ [ a , a≠0 ] ≡ [ b , b≠0 ])
+  where
+
+  private
+    module k = CommRingStr (snd k)
+    𝔸ⁿ⁺¹-as-module = FinVecLeftModule (CommRing→Ring k) {n = n ℕ.+ 1}
+    module 𝔸ⁿ⁺¹ = LeftModuleStr (str 𝔸ⁿ⁺¹-as-module)
+  open k using (_·_; -_; 0r; 1r)
+  open 𝔸ⁿ⁺¹ hiding (-_)
+
```
[the remark this answers is not in the document]
Comment (MatthiasHu) [reply SyntheticGeometry/ProjectiveSpace/LineThroughPoints.lagda.md:127]: I added something similar, thanks.

File: SyntheticGeometry/ProjectiveSpace/LineThroughPoints.lagda.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+    ((x , x≠0) : 𝔸ⁿ⁺¹-0 1)
+    where
+
+    x₀ = x zero
+    x₁ = x one
+
+    value : 𝔸ⁿ⁺¹ n
+    value = (x₀ ⋆ a) + (x₁ ⋆ b)
+
+    module _
```
[the remark this answers is not in the document]
Comment (MatthiasHu) [reply SyntheticGeometry/ProjectiveSpace/LineThroughPoints.lagda.md]: Good, done.

**Review by felixwellen — APPROVED** [review]

**Pull request merged**